docs/source/Advanced-Configuration.md

```diff
@@ -174,15 +174,41 @@ First, you need to configure the environment variable `LOCAL_STORAGE` to `False`
 Then you need to configure permission access to the chosen S3 bucket.
 
 
-#### SQS
+#### Message Broker
 
-If you like, you could use AWS SQS instead of Rabbit-MQ to manage your queues.
+IntelOwl at the moment supports 3 different message brokers:
+- RabbitMQ
+- Aws SQS
+- Redis
+
```

Comment:
considering that the websockets do not work if we don't have redis, can we really say that now?
I think there are 2 options:
* easiest one is to drop support for other brokers and release this change with the major 
* find a way to have the websocket work without redis but I already feel the pain of managing these 2 cases.

Reply:
Yes we can. 
There is nothing that prevents an user to use SQS and have redis for the websockets. Same thing with rabbitmq

Reply:
yes but in which case make sense to use another layer in the architecture when redis solve all of the problems at once? I'd remove the support for others, honestly

Reply:
I don't get why you want to remove the support since we have and it costs pretty much 0 to keep it. 
But again ,we can remove support (and add the breaking change in the next release)

Reply:
I checked again how the doc is written and the default configuration implemented cause I would like to do not create confusion on the user and yes, ok, I agree that by changing the doc a little we can maintain the support